src/assets/NcAppNavigationItem.scss

```diff
@@ -170,7 +170,7 @@
 	/* counter */
 	.app-navigation-entry__counter-wrapper {
 		// Add slightly more space to the right of the counter
```

Comment:
this will change the counter space, did you test it with the counter? if it looks off?

Reply:
![Bildschirmfoto_20240905_160729](https://github.com/user-attachments/assets/fa8774ee-df98-4d23-8e68-49f537ae816d)

2 and 4 have a counter, 3 has actions

Reply:
![Screenshot from 2024-09-05 16-39-13](https://github.com/user-attachments/assets/ebdedd46-4b92-4172-8a5c-e7c8e9b12a7c)

I would argue it looks better now though

Reply:
![image](https://github.com/user-attachments/assets/5005d89a-247e-47cb-8447-426bff7d747c)

If they are moved more to the right, the actions become misaligned with `NcAppNavigationCaption`, so unless you have any better ideas @susnux I would keep it the way it is currently

Reply:
That was no critic, just wanted to add screenshots for Gretas comment, I am fine with any of that :)